repair/row_level.cc

```diff
@@ -2195,6 +2195,13 @@ static future<> repair_put_row_diff_with_rpc_stream_handler(
                             current_rows,
                             std::move(row_opt)).handle_exception([from, repair_meta_id, sink, &error] (std::exception_ptr ep) mutable {
                         rlogger.warn("repair_put_row_diff_with_rpc_stream_handler: from={} repair_meta_id={} error={}", from, repair_meta_id, ep);
+                        if (try_catch<replica::no_such_column_family>(ep)) {
+                            return sink(repair_stream_cmd::table_dropped).then([sink] () mutable {
```

Comment:
This could happen with this RP:

- follower send repair_stream_cmd::table_dropped
- master exit the source() loop while more data can come 
- follower send repair_stream_cmd::put_rows_done

This should be ok because sink_source_for_repair would drain the source() when we close it. 

However, this "pretend" the repair is successful and handling error for no_such_column_family specially looks worrisome.